docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
 
-```xml
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeIdentifiers>win10-x64;osx.10.11-x64</RuntimeIdentifiers>
-  </PropertyGroup>
-</Project>
-```
+- `--self-contained false`  
+  This switch is used to tell the .NET Core SDK to create the executable (that targets the RID) as an FDE.
 
-## Self-contained deployment with third-party dependencies
+Whenever you use the `-r` switch, the output folder path changes to: `<CURRENT-FOLDER>/bin/<BUILD-CONFIGURATION>/<TFM>/<RID>/publish/`
 
-Deploying a self-contained deployment with one or more third-party dependencies involves adding the dependencies. Two additional steps are required before you can run the `dotnet restore` ([see note](#dotnet-restore-note)) command:
+If you were to use the [example app](#sample-app), run `dotnet publish -f netcoreapp2.2 -r win10-x64 --self-contained false`. This creates the following executable: `<CURRENT-FOLDER>/bin/Debug/netcoreapp2.2/win10-x64/publish/apptest1.exe`
 
-1. Add references to any third-party libraries to the `<ItemGroup>` section of your *csproj* file. The following `<ItemGroup>` section uses Json.NET as a third-party library.
+> [!Note]
+> You can reduce the total size of your deployment by enabling **globalization invariant mode**. This mode is useful for applications that are not globally aware and that can use the formatting conventions, casing conventions, and string comparison and sort order of the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture). For more information about **globalization invariant mode** and how to enable it, see [.NET Core Globalization Invariant Mode](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/globalization-invariant-mode.md)
 
-    ```xml
-      <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
-      </ItemGroup>
-    ```
+## Self-contained deployment
 
-1. If you haven't already, download the NuGet package containing the third-party dependency to your system. To make the dependency available to your app, execute the `dotnet restore` ([see note](#dotnet-restore-note)) command after adding the dependency. Because the dependency is resolved out of the local NuGet cache at publish time, it must be available on your system.
+If you publish a *self-contained deployment* (SCD), the .NET Core SDK creates a platform-specific executable, and adds all files needed to run the app to the `publish` folder. Publishing an SCD doesn't include the [native dependencies of .NET Core](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) on various platforms, so these dependencies must be present before the app runs. For more information on version binding at compile time, see [Select the .NET Core version to use](../versions/selection.md).
```

Comment:
- no comma before "and adds"
- but I'd reword this --> "platform specific executable and includes all files needed to run the app." "To the publish folder" is an unnecessary detail here.
- I'm not sure how the version binding sentence relates to this paragragh; a one-sentence explanation would be helpful. Also, presumably you want to link to an anchor (self-contained-deployments-include-the-selected-runtime) in the topic. Or do you want to link to the [Self-ontained deployment runtime roll forward](https://docs.microsoft.com/en-us/dotnet/core/deploying/runtime-patch-selection) topic?